go.mod

```diff
@@ -50,6 +50,9 @@ replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.
 // go-kit has the fix for nats-io/jwt/v2 merged but not released yet. Replacing this version for now until next release.
 replace github.com/go-kit/kit => github.com/go-kit/kit v0.12.1-0.20220808180842-62c81a0f3047
 
+// Use cadvisor main release when https://github.com/google/cadvisor/issues/2785 is fixed
+replace github.com/google/cadvisor => github.com/sethAmazon/cadvisor v0.44.1-0.20230221203935-b689186cd42f
```

Comment:
This turns on containerd containerfs metrics.

Reply:
I wonder if we can  using `docker` until 1.23 and using `containerd` later on with 1.24

Reply:
That is correct